src/main/java/service/operation/TransactionHandler.java

```diff
@@ -0,0 +1,7 @@
+package service.operation;
```

Comment:
It is better to move the package `operation` to the same level as the `service`
Then create a package `impl` inside, and to move there all handlers